extensions/functions_arithmetic.yaml

```diff
@@ -181,6 +211,72 @@ scalar_functions:
           - value: i64
           - value: i64
         return: i64
+  -
+    name: "power"
+    description: >
+      Take the power with the first value as the base and second as exponent.
+
+      All integer arguments are promoted to int64.
+      All floating point arguments are promoted to float64.
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i8
+          - value: i8
+        return: i64
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i16
+          - value: i16
+        return: i64
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i32
+          - value: i32
+        return: i64
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i64
+          - value: i64
+        return: i64
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: fp32
+          - value: fp32
+        return: fp64
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: fp64
+          - value: fp64
+        return: fp64
+  -
+    name: "sqrt"
```

Comment:
Same as power, sqrt is normally defined as `double sqrt(double)` and `float sqrtf(float)`. I suppose you could add `sqrt(i64) -> fp64` to those two because an `i64` can't be cast to fp64 without losing information, but the other ones are redundant.